### PR TITLE
Fix issue with CORS policy

### PR DESCRIPTION
**Pull request by federicoon**
Fixed HTTP request headers to prevent browsers from triggering CORS pre-flight requests

### CODE REVIEW

**Comment by lvaccaro** [comment]
In order to prevent the preflight request in browser (tested on Firefox 65.0), it requires to not overwrite the default 'User-Agent' of the browser, as explained in https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS .  
On the other hand, if the library is called by nodejs, you can send the 'User-Agent' to the server.
Could you make a check of the current environment in which the library is executed in order to set the 'User-Agent'?

**Comment by federicoon** [comment]
Added check on runtime environment, as requested

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ sync.sh
 *.wif
 deploy.sh
 dist/
+/.project
```
Comment (lvaccaro) [inline .gitignore]: this is not related to this topic

Comment (federicoon) [reply .gitignore]: removed unrelated change

File: src/calendar.js [hunk 2]
```diff
@@ -38,9 +38,12 @@ class RemoteCalendar {
     this.url = url
     this.headers = {
       Accept: 'application/vnd.opentimestamps.v1',
-      'User-Agent': 'javascript-opentimestamps',
       'Content-Type': 'application/x-www-form-urlencoded'
     }
+    if (!process.browser) { // running in node.js
+      console.log('Setting User-Agent in request header')
```
Comment (lvaccaro) [inline src/calendar.js]: the standard behaviour is to run ots library in nodejs with the right "User-Agent", so we don't need to provide any output. 

If you want to print a message only when you are in the browser, I suggest to revert the condition and instead of insert the user-agent, you could remove it with (keeping the previous headers): `delete this.headers['User-Agent']`

Comment (federicoon) [reply src/calendar.js]: removed unnecessary output message

**Comment by RCasatta** [comment]
merged f628d0d60a32a9e694cca74b2df2d3c2d107e20f
